### PR TITLE
added option to read pixel ratio and other cleanup

### PR DESCRIPTION
**Pull request by alinelena**

### CODE REVIEW

File: util.sh [hunk 1]
```diff
@@ -91,3 +91,56 @@ function buildmw {
     fi
     echo
 }
+function buildmwb {
```
Comment (sledges) [inline util.sh]: mwb?

File: dhd2modular.sh [hunk 2]
```diff
@@ -108,6 +100,9 @@ function migrate() {
         -e "s|@DEVICE_PRETTY@|$DEVICE_PRETTY|g" \
         -e "s|@VENDOR_PRETTY@|$VENDOR_PRETTY|g" \
         droid-configs-device/droid-config-@DEVICE@.spec.template >rpm/droid-config-$DEVICE.spec
+    read -r -p "Pixel Ration?[1.0,1.5 or 2.0]" PXR
```
Comment (sledges) [inline dhd2modular.sh]: Nitpicking: ```Select pixel ratio [1.0, 1.5, or 2.0]:```
Also shouldn't we check user entered only one of those three values and nothing else?

File: dhd2modular.sh [hunk 3]
```diff
@@ -108,6 +100,9 @@ function migrate() {
...
+    sed -i "s/\(pixel_ratio \)\(.*\)/\1$PXR/" rpm/droid-config-$DEVICE.spec
```
Comment (sledges) [inline dhd2modular.sh]: Let's do a stricter check and more readable:
```sed -i -e 's|^%define pixel_ratio .*$/%define pixel_ratio $PXR/' rpm/droid-config-$DEVICE.spec```

File: dhd2modular.sh [hunk 4]
```diff
@@ -216,6 +218,16 @@ function rpm_snapshot() {
...
 }
 
+function buildversion() {
+    cd hybris/droid-hal-version-$DEVICE
+    mb2 -t $VENDOR-$DEVICE-armv7hl \
+      -s rpm/droid-hal-version-$DEVICE.spec \
+      build
+    mv -v RPMS/*.rpm $LOCAL_REPO
+    cd ../../
+
```
Comment (sledges) [inline dhd2modular.sh]: extra newline

File: util.sh (hunk 1 above)
Comment (alinelena) [inline util.sh]: mw from branch... is not used at the moment is just a hidden gem that may help when debugging

File: dhd2modular.sh (hunk 2 above)
Comment (alinelena) [inline dhd2modular.sh]: That was my first thing in mind... but we already check inside the spec...

File: dhd2modular.sh (hunk 3 above)
Comment (alinelena) [inline dhd2modular.sh]: once in my life I get a sed command which is correct but unreadbale...

File: dhd2modular.sh [hunk 5]
```diff
@@ -108,6 +100,13 @@ function migrate() {
         -e "s|@DEVICE_PRETTY@|$DEVICE_PRETTY|g" \
         -e "s|@VENDOR_PRETTY@|$VENDOR_PRETTY|g" \
         droid-configs-device/droid-config-@DEVICE@.spec.template >rpm/droid-config-$DEVICE.spec
+    read -r -p "Pixel Ration?[1.0, 1.5 or 2.0]" PXR
+    case $PXR in
+       1.0|1.5|2.0) minfo "Good to go!"; ;;
+       *) merror "Pixel ratio must be 1.0, 1.5 or 2.0"; die; ;;
+    esac 
+    sed -i -e 's/^%define pixel_ratio .*$/%define pixel_ratio $PXR/' rpm/droid-config-$DEVICE.spec
```
Comment (sledges) [inline dhd2modular.sh]: Gah my bad! (and beer). Change to " " or $PXR would never get substituted :(

File: dhd2modular.sh [hunk 6]
```diff
@@ -108,6 +100,13 @@ function migrate() {
         -e "s|@DEVICE_PRETTY@|$DEVICE_PRETTY|g" \
         -e "s|@VENDOR_PRETTY@|$VENDOR_PRETTY|g" \
         droid-configs-device/droid-config-@DEVICE@.spec.template >rpm/droid-config-$DEVICE.spec
+    read -r -p "Pixel Ration?[1.0, 1.5 or 2.0]" PXR
+    case $PXR in
+       1.0|1.5|2.0) minfo "Good to go!"; ;;
+       *) merror "Pixel ratio must be 1.0, 1.5 or 2.0"; die; ;;
+    esac 
+    sed -i -e 's/^%define pixel_ratio .*$/%define pixel_ratio $PXR/' rpm/droid-config-$DEVICE.spec 
+    cat rpm/droid-config-$DEVICE.spec
```
Comment (sledges) [inline dhd2modular.sh]: Debug cruft? We all know the file -is- OK :D
And user will have to inspect all those files anyhow if some more logic is around

File: dhd2modular.sh (hunk 5 above)
Comment (alinelena) [inline dhd2modular.sh]: are you sure I have tested it and works without ""

File: dhd2modular.sh (hunk 6 above)
Comment (alinelena) [inline dhd2modular.sh]: ok I will get rid of it

**Comment by sledges** [comment]
LGTM

**Pull request merged**